packages/twenty-front/src/modules/object-record/cache/utils/getRecordNodeFromRecord.ts

```diff
@@ -45,16 +45,14 @@ export const getRecordNodeFromRecord = <T extends ObjectRecord>({
   }
 
   const nestedRecord = Object.fromEntries(
-    Object.entries(record)
-      .map(([fieldName, value]) => {
+    objectMetadataItem.fields
```

Comment:
agree that we should pilot the utils by the metadata and not the record sturcture but might have a lot of side effects in this app, we need to test everything then